app/gui2/src/stores/graph/index.ts

```diff
@@ -312,9 +312,15 @@ export const useGraphStore = defineStore('graph', () => {
     }
     const existingImports = readImports(topLevel)
 
+    // Let's check and see if the general import of the target module is already present.
+    const filteredNewImports = newImports.filter((newImport) => {
+      if (newImport.kind === 'Qualified') return true
+      return !existingImports.some((existingImport) => existingImport.from === newImport.from && existingImport.imported.kind === 'All')
+    });
+
```

Comment:
This should be done by working `detectImportConflicts` and `filterOutRedundantImports` below.

But the conflict resolution does not work properly; I would prefer removing the substitution from WidgetSelection, similar way as when creating node (https://github.com/enso-org/enso/blob/62c3a0f4f14c6f47e2f6a65ae5b83b245f724e81/app/gui2/src/stores/graph/index.ts#L292) Then we will fix it as part of https://github.com/enso-org/enso/issues/9356